pallets/identity/src/lib.rs

```diff
@@ -915,8 +916,8 @@ decl_error! {
         UnknownAuthorization,
         /// Account Id cannot be extracted from signer
         InvalidAccountKey,
-        /// Only KYC service providers are allowed.
-        UnAuthorizedKYCProvider,
+        /// Only CDD service providers are allowed.
+        UnAuthorizedCDDProvider,
```

Comment:
It makes sense to correct such occurrences to proper Pascal case, `Cdd`.

Reply:
My next commit will have the changes, thanks for the suggestion.